com.obsproject.Studio.Plugin.BackgroundRemoval.metainfo.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>com.obsproject.Studio.Plugin.BackgroundRemoval</id>
+  <extends>com.obsproject.Studio</extends>
+  <name>Background Removal</name>
+  <summary>Remove the background from your camera video</summary>
+  <description>Remove the background from your camera video without a green screen. Optimized for person portrait views.</description>
```

Comment:
This is bikeshedding and definitely not a blocker, but to me - a non-native English speaker, mind you! - `person portrait views` doesn't read well. What do you think of just `portrait mode`? Other options I can think of are: `portrait images`, `images in portrait mode`, or `portrait resolutions`.

Reply:
so, it's not "portrait mode" e.g. in the mobile sense of the orientation of the phone. the plugin removes the background when the camera view shows a portrait of a person. it works in any resolution or orientation of the image